### PR TITLE
Refactor Prisma call creating attendees and seat reference

### PR DESCRIPTION
**Pull request by joeauyeung**
## What does this PR do?

This PR aims to fix the bug on production where booking seats was failing to create due to required relation to attendees

https://threads.com/34459917807?s=xxqg1hjE5851xHMB3aw3ku

Fixes # (issue)

**Environment**: Staging(main branch) / Production

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- [ ] Test A
- [ ] Test B

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/features/bookings/lib/handleNewBooking.ts [hunk 1]
```diff
@@ -1236,14 +1236,9 @@ async function handler(
...
-        }),
-        await prisma.bookingSeat.update({
-          where: {
-            id: bookingSeat.id,
-          },
-          data: {
-            bookingId: newTimeSlotBooking.id,
+            bookingSeat: {
+              bookingId: newTimeSlotBooking.id,
+            },
```
Comment (joeauyeung) [inline packages/features/bookings/lib/handleNewBooking.ts:1241]: Is it worth also refactoring this Prisma call as well?

File: packages/features/bookings/lib/handleNewBooking.ts [hunk 2]
```diff
@@ -1298,32 +1295,25 @@ async function handler(
...
+                  data: {
+                    description: additionalNotes,
+                  },
+                  booking: {
+                    connect: {
+                      id: booking.id,
+                    },
+                  },
+                },
+              },
```
Comment (joeauyeung) [inline packages/features/bookings/lib/handleNewBooking.ts:1310]: Create the booking seat at the same time as we create the attendee

**Review by emrysal — APPROVED** [review]
LGTM :)

**Review by emrysal — CHANGES_REQUESTED** [review]
I can ignore the type error but we need to make sure this test passes:

image

**Review by emrysal — APPROVED** [review]

**Pull request merged**